spec/spec.md

```diff
@@ -1688,3 +1688,76 @@ Schema: [`TouchstoneOverview.schema.json`](TouchstoneOverview.schema.json)
             }
         }
     }
+
+# Demographics
+## GET /touchstones/{touchstone-id}/demographics/
+
+Returns a list of available demographic data sets for this touchstone, including for each data set
+whether multiple gender options are available.
+
+Required permissions: `demographics.read`.
+
+Schema: [`Demographics.schema.json`](Demographics.schema.json)
+
+### Example
+    [{ 
+        "code" : "as_fert"
+        "name": "Age-specific fertility",
+        "gender_is_applicable": false
+     },
+     { 
+        "code" : "tot_pop",
+        "name" : "Total population
+        "gender_is_applicable": true,
+     }]
+
+## GET /touchstones/{touchstone-id}/demographics/{demographic-type-code}/
+
+Returns the data set with given type, in `long` CSV format.
+
+### Example
+Age-specific fertility:
+    
+    "country",  "age of mother (years)",  "year", "avg births/mother"
+        "AFG",                  "15-19",    1950,                 1.2
+        "AFG",                  "15-19",    1955,                 1.2
+        "AFG",                  "15-19",    1960,                 1.2   
+        "AFG",                  "15-19",    1965,                 1.1   
+        "AFG",                  "15-19",    1970,                 1.1   
+        "AFG",                  "15-19",    1975,                 1.1  
+        "AFG",                  "15-19",    1980,                 1.1  
+        "AFG",                  "15-19",    1985,                 1.1  
+        "AFG",                  "15-19",    1990,                 1.1 
+        "AFG",                  "15-19",    1995,                 1.1 
+         
+### Example
+Total population:
+         
+     "country", "age (years)",  "gender",  "year",   "people"
+         "AFG",       "0-0",    "both",    1950,      82724
+         "AFG",       "0-0",    "both",    1951,      84699
+         "AFG",       "0-0",    "both",    1952,      87807
+         "AFG",       "0-0",    "both",    1953,      89014
+         "AFG",       "0-0",    "both",    1954,      89993
+                
+
+### Query parameters:
+
+#### gender
+Optional. The gender to return, either `female`, `male`, or `both`. Defaults to `both`.
+
+#### format
+Optional. A format to return the CSV in, either `wide` or `long`. Defaults to `long`.
+
+### Example
+`/touchstones/2017-op-1/demographics/tot_pop/?format=wide&gender=female`
+
+Total population:
+
+                                       "people"
```

Comment:
It looks like just a helpful header to say what the numbers mean (besides deriving it from the filename)

CSV is very relaxed - readers just have to skip the right number of header rows. You could prefix with "","","", to encourage it to line up with the first data column.

Source-wise, it's the "name" column of the demographic_value_unit table, where id matches the demographic_value_unit field in the demographic_statistic_type table.